### ISSUE TITLE
seb reset utility error when opening

### ISSUE THREAD

**Issue by LucasNiesen** [SafeExamBrowser/seb-win-refactoring#1029]
when i try to open seb reset utility i got this.
How can i fix this?
![image](https://github.com/user-attachments/assets/e60731ce-7585-4f4c-9056-edf52c7f23e8)

**Comment by dbuechel** [comment]
Is there a log file for the reset utility under `%LocalAppData%\SafeExamBrowser`? If yes, please do provide it. If not, please elaborate how exactly you're trying to start the reset utility and on which operating system version?

**Comment by dbuechel** [comment]
According to the error message, there is an issue with the buffer size of the console which you can configure in the console settings (if I recall correctly). Please try searching the internet for the given error message and report back here in case you cannot find any working solution.

**Comment by LucasNiesen** [comment]
> According to the error message, there is an issue with the buffer size of the console which you can configure in the console settings (if I recall correctly). Please try searching the internet for the given error message and report back here in case you cannot find any working solution.

Do you mean by 'console settings' cmd setting?

**Comment by dbuechel** [comment]
Yes, exactly.

**Comment by LucasNiesen** [comment]
I fixed it manualy. But should not be normal that the size of a terminal/console effects if a application starts or not.

**Comment by dbuechel** [comment]
That is correct. Please do elaborate what values you had which were not working and to what you did change them to make it work, so that we may improve the implementation to adapt to different console sizes.

**Comment by LucasNiesen** [comment]
> That is correct. Please do elaborate what values you had which were not working and to what you did change them to make it work, so that we may improve the implementation to adapt to different console sizes.

I had 166 and I needed to change it to 160. Then when I tried it again I got an error about height and needed to change it from 60 to 40.

**Comment by dbuechel** [comment]
Excellent, thanks for the information. We shall try to either fix that and/or at least provide a better user experience for an upcoming version.
